src/actions/tamanu/psql.rs

```diff
@@ -0,0 +1,112 @@
+use std::fs;
+
+use clap::Parser;
+use miette::{miette, Context as _, IntoDiagnostic, Result};
+use tracing::{debug, instrument};
+
+use crate::actions::Context;
+
+use super::config::{merge_json, package_config};
+use super::{find_tamanu, TamanuArgs};
+
+/// Connect to Tamanu's db via `psql`.
+#[derive(Debug, Clone, Parser)]
+pub struct PsqlArgs {
+	/// Package to look at
+	#[arg(short, long)]
+	pub package: String,
+
+	/// Include defaults
+	#[arg(short = 'D', long)]
+	pub defaults: bool,
```

Comment:
I think you've copied that from the config command but there's no need to include it here as we know that the db credentials will never be in defaults.

Reply:
The default config has `db.name`, which I use, so there's some benefit in pulling the default. I don't know if anyone needs to exclude the default config from this command though.

Reply:
Oh yep. Well, I think always pulling the defaults and not providing the option would be the right move here.